<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Терминал охранника</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RegButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-radius: 15px;
background-color: rgb(53, 132, 228);</string>
    </property>
    <property name="text">
     <string>Зарегистрировать сотрудника</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="card_number">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
color: rgb(255, 255, 255);
padding: 5px;
font: 25 italic 13pt &quot;Ubuntu&quot;;
border:  1px solid #fff;</string>
    </property>
    <property name="placeholderText">
     <string>Номер пропуска</string>
    </property>
   </widget>
   <widget class="QPushButton" name="access_employee_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(87, 227, 137);
font: 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Пропустить сотрудника</string>
    </property>
   </widget>
   <widget class="QPushButton" name="out_employee_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 51, 59);
font: 12pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Выпустить сотрудника</string>
    </property>
   </widget>
   <widget class="QPushButton" name="temp_card_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: rgb(154, 153, 150);
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>Выдать временный пропуск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parking_ticket_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-radius: 15px;
background-color: rgb(154, 153, 150);
</string>
    </property>
    <property name="text">
     <string>Выдать парковочный талон</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
